### PR TITLE
Fix lint errors in readme and alter workflow checks

### PR DESCRIPTION
**Pull request by Jaydee123**

### CODE REVIEW

File: .github/workflows/most_recent_pr_iqengineweb.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
           path: client/build/
 
   deploy-to-most-recent-pr-slot:
-    if: github.repository_owner == 'IQEngine'
+    if: github.event.pull_request.head.repo.full_name == 'IQEngine/IQEngine'
```
Comment (Nepomuceno) [inline .github/workflows/most_recent_pr_iqengineweb.yml:45]: SHouldn't we move this to the top of the JOb there is no point on doing anything of this job if this is the case.

**Review by Nepomuceno — COMMENTED** [review]
My understanding it is that this will stop any PR that does not come from the same repo that IQEngine is ( this menas any fork ) not to deploy to the latest pr slow if this is what we are thinking about doing perfect

File: .github/workflows/most_recent_pr_iqengineweb.yml (hunk 1 above)
Comment (Jaydee123) [reply .github/workflows/most_recent_pr_iqengineweb.yml:46]: Conditionals work at job level so have added to other job also:
https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution

**Review by Nepomuceno — APPROVED** [review]
:shipit:

**Pull request merged**